### PR TITLE
Fix #6

### LINKED ISSUE #6: Add thumbnail download
Currently, the thumbnails for the cards are fetched using the Makefile for this project.  The advantage for this is that you can separate some of the tasks that go out to trello from the task of creating the moment.  This isn't a good final solution however; it'd be better to include this in the trello2moment application with a flag on whether to fetch it or not.

### PR DESCRIPTION
**Pull request by ldragoon**

### CODE REVIEW

File: trello2moment [hunk 1]
```diff
@@ -69,9 +69,9 @@ Object.keys(schema).forEach((k)=>{schema[k]=namedNode(prefix.schema+k)});
 program
 		.version('0.5.0')
 		.description('Convert a Trello board into a UCDavis library moment linked dataset.')
-		.option('-o, --output_file <output_file>','JSON output (default {moment}/moment.json)',null)
-    .option('-i, --images','create board images',false)
-    .option('--overwrite','overwrite existing <board>.json',false)
+    .option('-v, --verbose','verbose output',false)
```
Comment (qjhart) [inline trello2moment:72]: Added Verbose Option

Comment (qjhart) [inline trello2moment:72]: Added --verbose

File: trello2moment [hunk 2]
```diff
@@ -85,13 +85,28 @@ function add(s,p,o) {
 	return writer.addQuad(s,p,o);
 }
 
+// Allow Verbose Output
+function log(text) {
```
Comment (qjhart) [inline trello2moment:89]: Added Log() function for --verbose

File: trello2moment [hunk 3]
```diff
@@ -85,13 +85,28 @@ function add(s,p,o) {
...
+  if (program.verbose) {
+    console.log(text)
+  }
+}
+
+// Check for file, but not if overwrite
+function fileExists(fn) {
```
Comment (qjhart) [inline trello2moment:96]: Added --overwrite test for images

File: trello2moment [hunk 4]
```diff
@@ -116,7 +131,62 @@ function getBoard(boardPath) {
...
+  for ( let id in jsonObj ) {
+    if ( jsonObj[id].cover.idAttachment != null ) {
+      let shortId = jsonObj[id].shortLink;
+      for ( let item of jsonObj[id].attachments ) {
+        if ( item.isUpload ) {
+          let d=path.join(momentName, shortId);
+          if ( !fs.existsSync(d) ) {
+            log(`mkdir ${d}(.ttl)`)
+            fs.mkdirSync(d);
+            let ttl = new N3.Writer({prefixes: prefix});
```
Comment (qjhart) [inline trello2moment:144]: Added directory TTL here

File: trello2moment [hunk 5]
```diff
@@ -116,7 +131,62 @@ function getBoard(boardPath) {
...
+function fetchFile(url,fileName) {
+  if ( ! fileExists(fileName) ) {
+    log(`fetch ${url} to ${fileName}`);
+    return new Promise(async (resolve, reject) => {
+      try {
+        const res = await fetch(url);
+        res.body
+          .pipe(fs.createWriteStream(fileName))
+          .on('close', () => resolve());
+        let ttl = new N3.Writer({prefixes: prefix});
```
Comment (qjhart) [inline trello2moment:169]: Added Image file .ttl file here

File: trello2moment [hunk 6]
```diff
@@ -148,34 +213,19 @@ async function moment (key, token, moment_name, board_name, output_file, images,
...
 
   writer.end((error, result) => {
-    if (program.output_file) {
-      fs.writeFile(program.output_file, result, (err) => {
-        if (err) throw err;
-      });
-    } else {
-      console.log(result);
-    }
+    fs.writeFile(program.moment+'_moment.ttl', result, (err) => {
```
Comment (qjhart) [inline trello2moment:226]: Removed --output-file, and always write to the same place

File: trello2moment (hunk 3 above)
Comment (ldragoon) [reply trello2moment:96]: Got it! :)

File: trello2moment (hunk 4 above)
Comment (ldragoon) [reply trello2moment:144]: Got it! :)

File: trello2moment (hunk 5 above)
Comment (ldragoon) [reply trello2moment:169]: Got it! :)

File: trello2moment (hunk 6 above)
Comment (ldragoon) [reply trello2moment:226]: Got it! :)

File: trello2moment (hunk 2 above)
Comment (ldragoon) [reply trello2moment:89]: Got it! :)

**Pull request merged**